docs/tutorials/python/wiki.md

```diff
@@ -1,2 +1,152 @@
 # Wikis on Projects
-![Under Construction](../../assets/under_construction.png)
+
+# Synapse Wiki Models Tutorial
+
+This tutorial demonstrates how to work with Wiki models in the Synapse Python client. Wikis in Synapse provide a way to create rich documentation and collaborative content for projects, folders, files, datasets, and other entities.
+
+## Overview
+
+The Synapse Wiki models include:
+- **WikiPage**: The main wiki page model for creating and managing wiki content
+- **WikiHeader**: Represents wiki page headers and hierarchy information
+- **WikiHistorySnapshot**: Provides access to wiki version history
+- **WikiOrderHint**: Manages the order of wiki pages within an entity
+
+This tutorial shows how to:
+1. Create, read, and update wiki pages
+2. Work with WikiPage Markdown
+3. Work with WikiPage Attachments
+4. Work with WikiHeader
+5. Work with WikiHistorySnapshot
+6. Work with WikiOrderHint
+7. Delete wiki pages
+
+## Basic Setup
+```python
+{!docs/tutorials/python/tutorial_scripts/wiki.py!lines=14-29}
+```
+
+## 1. Create, read, and update wiki pages
+### Create a new wiki page for the project with plain text markdown
+```python
+{!docs/tutorials/python/tutorial_scripts/wiki.py!lines=33-37}
+```
+
+### OR you can create a wiki page with an existing markdown file
+```python
+{!docs/tutorials/python/tutorial_scripts/wiki.py!lines=40-45}
+```
+
```

Comment:
A great addition would be to include a section that can be expanded to show what the console logs would be after running a particular step.

There are some examples on other tutorials that can be followed